### DISCUSSION TITLE
Firefly Password not set and unable to Stake

### DISCUSSION THREAD

**Discussion by vikky4446** [iotaledger/firefly#2960] [Help & Support]
Hi,

I haven't set my password for Firefly. I'm trying to stake my Miota for Assembly. I am prompted to enter password. I haven't set any password previously and I don't see any other option to proceed for staking.

Kindly help
![Firefly error](https://user-images.githubusercontent.com/104023549/164058012-299799f8-1c6f-4afd-813a-d9b8b6efbf6c.JPG)
!

**Comment by MiKeF05** [comment]
You have set this password when you created the profile. It is needed to unlock your wallet. You can also create a new profile and choose recover wallet with Firefly backup. Then you can restore it with your 24 words and choose a new password
